server/src/modules/user/user.controller.ts

```diff
@@ -45,7 +45,7 @@ import { MapDto } from '@common/dto/map/map.dto';
 import { MapFavoriteDto } from '@common/dto/map/map-favorite.dto';
 import { MapLibraryService } from '@modules/maps/map-library.service';
 
-@Controller('api/user')
+@Controller('user')
```

Comment:
This isn't the right fix for the tests 404ing - I'll have that in shortly. So please leave this out of the commit, if you want to fix everything in the meantime, just remove L42 `app.setGlobalPrefix('api', { exclude: ['auth'] });` from `tests/e2e/environment.ts`.